### ISSUE TITLE
Local Mapping vs. Pointiness in Material

### ISSUE THREAD

**Issue by provisory** [LuxCoreRender/LuxCore#221]
#### System Information

Win10, CPU

#### Software Version

* Blender version: 2.79
* LuxCore version: luxcorerender-latest-win64 - 07/08/2019
* BlendLuxCore version: BlendLuxCore-2_79_maintenance - 06/25/2019

#### Error message from Blender console

No error message

#### Short description of error

It seems, that Local 3D Mapping and Pointiness doesn't work together in materials.

#### Exact steps for others to reproduce the error 

When I use Local mapping for procedural texture in 3D Mapping node without Pointiness node it works well both in the viewport and in rendering. 
But if I add a Pointiness node (it doesn't even has to connect to anything!), the rendering result goes wrong, like the 3D Mapping was on Global setting instead of Local. 
The viewport still shows correct result.

If I write LuxCore scene from Blender, the last line of the .scn file is different depending on whether there is the Pointiness node.

Without Pointiness:
```
scene.objects.Mesh_Cube_1844402002280000.appliedtransformation = -0.610595226 0.599808574 0.517110407 0 -0.0484430008 0.623453081 -0.780358553 0 -0.790459871 -0.501533568 -0.351620942 0 0 0 0 1
```

With Pointiness:
```
scene.objects.Mesh_Cube_1844402002280000.appliedtransformation = 1 0 0 0 0 1 0 0 0 0 1 0 0 0 0 1
```

Viewport with Pointiness node:
![LocalMappingVsPointiness-viewport](https://user-images.githubusercontent.com/45516148/60903556-dd574900-a271-11e9-9d65-9b89f512f41c.jpg)

Rendering with Pointiness node:
![LocalMappingVsPointiness-render](https://user-images.githubusercontent.com/45516148/60903557-dd574900-a271-11e9-8218-84ee5c7079c1.jpg)

[LocalMappingVsPointiness.zip](https://github.com/LuxCoreRender/BlendLuxCore/files/3373851/LocalMappingVsPointiness.zip)

**Comment by Theverat** [comment]
I pushed a fix, maybe @Dade916 can check if I solved it correctly.

**Comment by provisory** [comment]
The error no longer occurs with the new build.

Thank you!
